subprojects/docs/src/docs/userguide/migration/upgrading_version_7.adoc

```diff
@@ -373,6 +373,14 @@ In Gradle 8.0, `java-base` handles this configuration.
 No changes are required for projects that already apply the
 `java-base` plugin directly or indirectly through the `java`, `application`, `java-library`, or other JVM ecosystem plugins.
 
+==== Toolchain-based tasks for JVM projects
+
+Starting with Gradle 8.0 all core Java tasks that allowed configuring toolchains are now using them unconditionally.
```

Comment:
```suggestion
Starting with Gradle 8.0, all core Java tasks that allowed configuring toolchains are now using them unconditionally.
```

Reply:
Other suggestion:
> Starting with Gradle 8.0, all core Java tasks that have toolchain support are now using them unconditionally.

I hesitate between "support" and "integration"